bentoml/_internal/frameworks/pytorch.py

```diff
@@ -43,8 +33,6 @@
         """  # noqa
     )
```

Comment:
I don't think we need this block anymore since you already do an import check under `.common.pytorch`

Reply:
I'm thinking about just keeping these blocks in case one day we remove torch import from `.common.pytorch`. What's your opinion? @aarnphm

Reply:
Hmm why would we do so? I don't think we would ever do such things right?

Reply:
![Screenshot 2022-02-26 at 17 43 02](https://user-images.githubusercontent.com/29749331/155861366-3e3f6722-ee50-45ea-abcd-8b64cfa66f0e.png)

Reply:
I made a change to import torch from .common.pytorch directly @aarnphm